regulations/unified_metrics/environment/biodiversity_sensitive_sites.md

```diff
@@ -55,15 +69,17 @@ biodiversity
 
 ### value_question_en
 
-
+Does the company have operations in or near biodiversity sensitive areas? If so, please indicate the total number of sites and
+size of area in hectares.
 
 ### value_question_fr
 
 
 
 ### summary_en
 
-
+The undertaking shall disclose the number and area (in hectares) of sites that it owns, that it has leased or that it manages,
+that are located in or near biodiversity sensitive areas.
```

Comment:
Question should start more like:
- The company should disclose the sites... it owned ... (in the past)
and finish with:
- , during the period ?